docker-files/database/init-files/test_mysql_init.sql

```diff
@@ -121,10 +191,16 @@ CREATE TABLE "customers" (
 	"created_at" TIMESTAMP,
 	"updated_at" TIMESTAMP,
 	PRIMARY KEY("customer_id")
-);
+)
+INSERT INTO "customers" ("customer_id","name","email","prefecture","city","address","post_code"."password_hash","created_at","updated_at") VALUES
+('fdd25989-085e-fc11-31f0-a4a25095a47d','波留太郎','sample1@gmail.com','北海道','札幌市','1-23-4','893-1101','9f738ce8457f291b18ee47e665e96baa84f38fcd','2024-11-5-12:00','2024-11-5 12:00'),
```

Comment:
```suggestion
('fdd25989-085e-fc11-31f0-a4a25095a47d','波留太郎','sample1@gmail.com','北海道','札幌市','1-23-4','8931101','9f738ce8457f291b18ee47e665e96baa84f38fcd','2024-11-5-12:00','2024-11-5 12:00'),
```

これはテストデータ側の不備だけど！
ハイフン消してもらえると！